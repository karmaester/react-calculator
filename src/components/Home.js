const Home = () => (
  <div className="welcome">Welcome to Math Magicians</div>
);

export default Home;
