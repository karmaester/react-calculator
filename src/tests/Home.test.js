import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/Home';

test('should render the Home Page', () => {
  const component = renderer.create(
    <BrowserRouter>
      <Home />
    </BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
