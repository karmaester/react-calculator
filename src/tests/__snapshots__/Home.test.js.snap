// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Home Page 1`] = `
<div
  className="welcome"
>
  Welcome to Math Magicians
</div>
`;
